PrivateCollections/Ethiopia/MentasnotAbbuhay/MEAG001.xml

```diff
@@ -0,0 +1,497 @@
+                              <ab type="ruling">The upper line is written above the ruling,
+                                 the lower line is written above the ruling.</ab>
+                              <locus from="5r" to="183v"></locus>
+                           </layout>
+                           
+                           
+                           <layout columns="2" writtenLines="23">
+                              <locus from="184ra" to="196rb"/>
+                              <ab type="pricking">Pricking is visible.</ab>
+                              <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                              <ab type="ruling">Ruling is visible.</ab>
+                              <ab type="ruling">The upper line is written above the ruling,
+                                 the lower line is written above the ruling.</ab>
+                           </layout>
+                        </layoutDesc>
+                     </objectDesc>
+                     
+                     <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                           <seg type="script">Twentieth century script.</seg>
+                           <seg type="ink">Black, red</seg>
+                           <date notBefore="1900" notAfter="1950"/>
+                           <desc>Regular, fine; by a professional scribe.</desc>
+                           <persName ref="PRS14379Abbuhay" role="scribe"/>
+                        </handNote>
+                     </handDesc>
+                     
+                     <decoDesc>
+                        <decoNote type="miniature" xml:id="d1">
+                           <locus target="#3v"/>
+                          <desc>A miniature depicting <ref type="authFile" corresp="AT1066DavidHarp"> the biblical king David playing harp</ref>.
+                           </desc>
+                        </decoNote> 
+                        
+                        
+                        <decoNote type="band" xml:id="d2">
+                           <locus target="#5r #14r #24r #34r #47r #57r #66r #78r #92r #103r #111r #127r #143r #147r #154r #161r #175r "/>
+                           <desc>Harags are decorated with different colors and flowers after every ten Psalms.</desc>
+                        </decoNote>
+                        
+                        <decoNote type="miniature" xml:id="d3">
+                           <locus target="#196v"/>
+                           <desc>A miniature depicting <ref type="authFile" corresp="AT1031VirginAndChild"/>:
+                              <persName ref="PRS6819Mary"></persName> with the
+                              Child in the center of the image, at the top protected by two angels, and at the bottom protected by two male figures 
+                              to the left and right (it is probabel that this two figures depict <persName ref="PRS3827Ephrem"/> 
```

Comment:
'probable' or better: 'likely'. Personally, I prefer to give additional information in a new sentence without brackets.